### PR TITLE
Allow delete when acknowledge

### PR DESCRIPTION
**Pull request by nicoalonso**
### Implementation

An option is added to delete the message when it was acknowledge.

### CODE REVIEW

File: lib/interfaces.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ interface RedisStreamOptionsXreadGroup {
   block?: number;
   consumerGroup: string;
   consumer: string;
+  deleteAfterAck?: boolean;
```
Comment (tamimaj) [inline lib/interfaces.ts:18]: I think `deleteMessagesAfterAck` is a better name.

Comment (nicoalonso) [reply lib/interfaces.ts:18]: Ok for me. I copy from Symfony Messenger.

File: README.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ async function bootstrap() {
         block: 5000,
         consumer: 'users-1',
         consumerGroup: 'users',
+        deleteAfterAck: true,  // optional: delete message from stream
```
[the remark this answers is not in the document]
Comment (nicoalonso) [reply README.md:100]: Yes, I forgot

**Comment by tamimaj** [comment]
@nicoalonso Thanks, I will merge and publish!

**Review by tamimaj — APPROVED** [review]

**Comment by nicoalonso** [comment]
@tamimaj Thanks

**Comment by tamimaj** [comment]
@nicoalonso is published. v1.0.3!
Thank you very much!

**Pull request merged**